packages/api/src/modules/file/resolvers.ts

```diff
@@ -0,0 +1,72 @@
+import { FileUpload } from 'graphql-upload';
+
+import { model as File } from '.';
+import { model as Project } from '../project';
+import { MAX_ALLOWED_FILE_SIZE_IN_BYTES } from './constants';
+
+interface ICreateFileArgs {
+  file: FileUpload;
+  projectId: string;
+  sourceLanguage: string;
+}
+
+interface IUpdateFileArgs {
+  newFilename: string;
+  projectId: string;
+  fileId: string;
+}
+
+async function createFile(parent, args: ICreateFileArgs, context) {
+  const { createReadStream, filename } = await args.file;
+
+  if (context.contentLength > MAX_ALLOWED_FILE_SIZE_IN_BYTES) {
+    throw new Error('File size exceeded, limit is 5MB.');
+  }
+
+  // const stream = createReadStream();
+  // stream.on('data', (chunk) => console.log(chunk.toString()));
```

Comment:
Apagar comentário ou vai ser importante depois?

Reply:
qndo a gente for usar a gente faz de novo